torch/testing/_internal/distributed/distributed_test.py

```diff
@@ -5778,7 +5778,31 @@ def test_broadcast_object_list(self):
                 else [None for _ in COLLECTIVES_OBJECT_TEST_LIST]
             )
 
-            # Single object test
+            # Single object test with device specified. Backend="gloo", device=cpu
+            if backend == "gloo":
```

Comment:
Curious, will `backend != "nccl"` work here? If so, that might be better, as it will also give us coverage for MPI backend.

Reply:
in the header 
@require_backend({"nccl", "gloo"}) is specified
does this mean only nccl and gloo backends are valid for this test?

anyway, changing it to backend != nccl should work